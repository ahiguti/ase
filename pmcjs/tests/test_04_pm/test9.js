
use('Perl');
Perl.Use('File::Copy');
var fops = Perl.Functions['File::Copy'];
var stdout = Perl.Handles[''].STDOUT;
if (!fops.copy('abc.txt', stdout)) {
  print("FAILED");
} else {
  print("copied from abc.txt to stdout");
}

