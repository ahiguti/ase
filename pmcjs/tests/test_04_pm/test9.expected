this is abc.txt
copied from abc.txt to stdout
